react-app/src/components/projects/ProjectDetail.tsx

```diff
@@ -293,6 +320,28 @@ const ProjectDetail = (props: IProjectDetail) => {
           onEdit={() => setOpenProjectInfoDialog(true)}
           disableEdit={!isAdmin}
         />
+        {hasERPNotifications && (
+          <DataCard
+            values={undefined}
+            loading={isLoading}
+            title={enhancedReferralDates}
+            id={enhancedReferralDates}
+            onEdit={() => {}}
+            disableEdit={true}
+          >
+            <EnhancedReferralDates
+              rows={
+                notifications?.items
+                  ? notifications.items.map((resp) => ({
+                      AgencyName: lookup.getLookupValueById('Agencies', resp.ToAgencyId)?.Name,
+                      ChesStatusName: getStatusString(resp.Status),
```

Comment:
It looks like these extra mapped properties aren't being used in the component.
This could just be
```
<EnhancedReferralDates rows={notifications?.items} />
```